app/controllers/api/v1/resources/tickets_controller.rb

```diff
@@ -25,4 +25,9 @@ def create
     end
   end
 
+  def show
+    ticket = Ticket.find(params["ticket_id"].to_i)
+    render json: ticket
```

Comment:
This reminds me (from the depths of my rusty Rails brain) eventually we'll want serializers here?